src/main/java/uk/ac/ebi/ega/egafuse/runner/CommandLineOptionParser.java

```diff
@@ -52,10 +52,7 @@ public static CliConfigurationValues parser(OptionSet optionSet) throws IOExcept
         if (optionSet.has("cf")) {
             cliConfigurationValues.setCredential(readCredentialFile((Path) optionSet.valueOf("cf")));
         } else {
-            Credential credential = new Credential();
-            credential.setUsername(optionSet.valueOf("u").toString());
-            credential.setPassword(optionSet.valueOf("p").toString());
-            cliConfigurationValues.setCredential(credential);
+            cliConfigurationValues.setCredential(readCredentialConsole());
```

Comment:
I thought the task was to remove the -u and -p options only from the fuseclient.sh script, but leave those options in the Java application (maybe for the "power users"?).

Reply:
Ideally, this should be confirmed with Dylan. I think he suggested to remove the -u and -p options only from the bash script, but he also suggested to keep it in the Java application. But I might be wrong.